src/html.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function HTML({
+  body,
+  htmlAttributes,
+  headComponents,
+  bodyAttributes,
+  preBodyComponents,
+  postBodyComponents
+}) {
+  return (
+    <html lang="en" {...htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {headComponents}
+      </head>
+      <body {...bodyAttributes}>
+        {preBodyComponents}
+        <noscript key="noscript" id="gatsby-noscript">
+          This app works best with JavaScript enabled.
+        </noscript>
+        <div className="preloader"></div>
```

Comment:
I'm assuming we're adding this file _just_ to insert this `div` here. Can't we add the div in the `Page` component instead: https://github.com/CodeForAfrica/sensors.AFRICA-SeaSensors/blob/e414a8b662e94765fa10391dd6ccd88676eecc2c/src/components/core/Page.js#L10

All pages already use this component so if it works here, it should be available in all pages.

Reply:
@kilemensi Maybe a small update on this...putting the` preload ` page.js loads it at the same time with everything else on the page... And we need to catch the `::before ` the page loads for us to load the preloader spinner. Not sure if you have any suggestions on the way forward?